app/components/Table/TableContent/TableContent.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Flex from '../../Flex/Flex';
+import props from '../../../../interfaces/props';
+import Link from 'next/link';
+const TableContent: React.FC<props.TableContentProps> = ({ player }) => {
+  const { id, firstname, lastname, goal, salary, devise } = player;
+  const deletedPlayer = () => {
```

Comment:
- It is better to separate all network calls to a sperate folder like services to keep the component more readable, and also the link of the fetch shouldn't be hardcoded, it's best to use it as an env variable.
- The Fetch method doesn't catch errors , it would be better to handle fetch errors and provide feedback to the user if the request fails